xds/src/test/java/io/grpc/xds/orca/OrcaOobUtilTest.java

```diff
@@ -153,7 +153,7 @@ private static OrcaLoadReportRequest buildOrcaRequestFromConfig(
   }
 
   private static void assertLog(List<String> logs, String expectedLog) {
-    assertThat(logs).containsExactly(expectedLog);
+    assertThat(logs).contains(expectedLog);
```

Comment:
This might be a questionable change, but I think this is the better way to test if a log message is present. However, I can rework this if needed.